tap_s3_csv/__init__.py

```diff
@@ -13,7 +14,7 @@
 from tap_s3_csv.sync import sync_stream
 from tap_s3_csv.config import CONFIG_CONTRACT
 
-LOGGER = get_logger('tap_s3_csv')
+LOGGER = get_logger()
```

Comment:
We can't remove the logger name, it's important to keep in order to identify which component emits which log line. This is crucial to have when running this plugin with Pipelinewise.